SConstruct

```diff
@@ -779,6 +788,8 @@ noshell = has_option( "noshell" )
 
 jsEngine = get_option( "js-engine")
 
+rocksdb = get_option( "rocksdb" ) == "on"
```

Comment:
I've always used `--rocksdb=1` with v3.2 to build PSMDB with rocksdb support. I think Christian suggested this to me a long time ago.
Probably the same form is used in David's scripts. So if new code will only accept `--rocksdb=on` then probably David will need to modify his scripts.
So I think compatibility-wise it is safer to support both forms.